src/main/java/pp/javastart/budget/TransactionDao.java

```diff
@@ -0,0 +1,113 @@
+package pp.javastart.budget;
+
+import java.sql.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionDao {
+
+    public void insert(Transaction transaction) {
+
+        Connection connection = connect();
+
+        PreparedStatement preparedStatement = null;
+        try {
+            String sql = "INSERT INTO transactions(type, description, amount, date) VALUES (?, ?, ?, ?)";
```

Comment:
To można na spokojnie wynieść do stałej - nie zmienia się podczas całego działania programu.

Reply:
done